xplt/line.py

```diff
@@ -336,10 +349,22 @@ def update(self, survey=None, line=None, *, autoscale=None):
             # ang: transform angles from data (A-B) to axis (X-Y) coordinate system
             if self.projection == "ZX":
                 R = get(survey, "theta")
-                ang = lambda a: a
+
+                def ang(a):
+                    return a
+
             elif self.projection == "XZ":
                 R = get(survey, "theta")
-                ang = lambda a: np.pi / 2 - a
+
+                def ang(a):
+                    return np.pi / 2 - a
+
+            elif self.projection == "ZY":
+                R = get(survey, "theta")
```

Comment:
This doesn't look right, ZY is a sideways view, the correct angle should be phi, isn't it?

Reply:
Yes indeed! This slipped in by mistake. I 'll make a PR